package.json

```diff
@@ -4,9 +4,27 @@
   "main": "index.js",
   "license": "MIT",
   "scripts": {
-    "routine": "node routine.js"
+    "routine": "node ./src/routine.js",
+    "availability": "env $(print-env) node ./src/availability",
```

Comment:
this won't work without the commented out logic in `availability.js`. see comment there